package.json

```diff
@@ -189,7 +190,7 @@
     "build:js": "tsc --p tsconfig.build.json",
     "build:copy-assets": "cd src/lib && find . -name '*.scss' -o -name '*.css' -o -name '*.svg' | cpio -pdm ../../dist/",
     "build:rollup": "rollup -c && cp -r src/umd dist/",
-    "serve:docs": "npx styleguidist server --open",
+    "serve:docs": "DISABLE_ESLINT_PLUGIN=true npx styleguidist server --open",
```

Comment:
By default, ESLint runs before the styleguidist server launches. This can take a long time (seemingly longer with CRA v4/ESLint v7), and it's not very useful since we aren't actively monitoring/using our lint rules at this time.

So this effectively speeds up `yarn start`. We can change this if we decide to spend time configuring our linter and cleaning up existing lint issues.